plugins/optimization-detective/class-od-url-metric-group-collection.php

```diff
@@ -193,6 +193,39 @@ public function get_current_etag(): string {
 		return $this->current_etag;
 	}
 
+	/**
+	 * Gets the breakpoints in max widths.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return positive-int[] Breakpoints in max widths.
+	 */
+	public function get_breakpoints(): array {
+		return $this->breakpoints;
+	}
+
+	/**
+	 * Gets the sample size for URL Metrics for a given breakpoint.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return int<1, max> Sample size for URL Metrics for a given breakpoint.
+	 */
+	public function get_sample_size(): int {
+		return $this->sample_size;
+	}
+
+	/**
+	 * Gets the freshness age (TTL) for a given URL Metric..
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return int<0, max> Freshness age (TTL) for a given URL Metric.
+	 */
+	public function get_freshness_ttl(): int {
+		return $this->freshness_ttl;
+	}
```

Comment:
Ideally this would be in its own PR, since it is completely unrelated and won't appear in changelogs if it's included here. Doesn't have to be changed now, but would be good to keep PRs separated by purpose.

Reply:
It's not entirely unrelated as it involves the freshness TTL. I didn't think it warranted a separate changelog entry so that's why I included it here.